### DISCUSSION TITLE
🤔 Plan

### DISCUSSION THREAD

**Discussion by recoluan** [vuepress-reco/vuepress-theme-reco#2] [Show and tell]
reco 主题 2.0，依赖 `@vuepress-reco/vuepress-plugin-page` 为主题主题扩展的页面，同时可以为特定页面注入分类、标签和分页的相关数据，并依赖其他插件默认为主题扩展一些必要功能。

**Comment by ifzm** [comment]
支持！
话说大佬考虑以后用 vitepress 么 ( ˘•ω•˘ )

**Reply by recoluan** [reply]
目前 vitepress 只是大佬的玩具，等到后面成熟了再支持。

另外 vuepress@2.x 支持切换编译工具至 vite，可以看一下官方文档。

**Reply by ifzm** [reply]
好的

**Comment by KimYangOfCat** [comment]
点赞，不知道大佬有没有想法支持**在线简历**这种风格？

**Reply by recoluan** [reply]
不错的建议，采纳。
